### ISSUE TITLE
Attach autocomplete jquery function to input text

### ISSUE THREAD

**Issue by Ant02** [free-jqgrid/jqGrid#320]
Hi Oleg,  there is any way to do attach the jquery autocomplete functionality to all textbox in edit mode? 

For example in this demo(https://jsfiddle.net/01th999n/10/) I need to attach the autocomplete js for ID name and address, there is a stadard way to do that?

**Comment by OlegKi** [comment]
Look at the demo https://jsfiddle.net/OlegKi/yvbt6w54/1/, which I included in the README of free jqGrid 4.14.0 (see [New main features implemented in the version 4.14.0 compared with 4.13.6](https://github.com/free-jqgrid/jqGrid#new-main-features-implemented-in-the-version-4140-compared-with-4136)). It shows how to use jQuery UI Autocomplete (and select2) in combination with `createColumnIndex: true`. If you want to apply new settings for all columns, the you can place additional properties in `cmTemplate`: 
```
cmTemplate: {
    editable: true,
    autoResizable: true,
    createColumnIndex: true,
    editoptions: {
        dataInit: function (elem, options) {
            $(elem).autocomplete({
                source: $(this).jqGrid("getUniqueValueFromColumnIndex", options.name),
                delay: 0,
                minLength: 0
            });
        }
    }
}
```
See https://jsfiddle.net/o8xLvbuz/

**Comment by Ant02** [comment]
Perfect thanks
